### DISCUSSION TITLE
Add Velero to the default installation to provide backup capabilities

### DISCUSSION THREAD

**Discussion by danielpsf-grover** [kubero-dev/kubero#91] [Ideas/Features]
As a Platform Engineer, I'd like to have kubero.dev installation package to come with a default installation of velero.io so that I can use the UI to configure which kind of backup routine I'd like to set for my apps and their components, such as databases and all.

**Comment by mms-gianni** [comment]
I like this idea. (Having a backup is always a good idea) . But to be honest, I'm not very familiar with velero.io . It will take some time to get into this topic. 

This might also be a possibility to initialize data for a pull request instance of an app.
